data/ui/book_card.blp

```diff
@@ -132,6 +132,11 @@ menu book_menu {
       action: 'app.jump_to_book_folder';
       label: _("Open in Files");
     }
+
+    item {
+      action: 'app.reset_book';
+      label: _("Remove from Recents");
+    }
```

Comment:
I think this item would be better above "Open in Files". It's more related to "Mark as Read" as bot of these options effect the book state.